experimental/torch_xla2/torch_xla2/__init__.py

```diff
@@ -10,19 +10,30 @@
   'extract_jax',
 ]
 
-
+from jax._src import xla_bridge
+os.environ.setdefault('ENABLE_RUNTIME_UPTIME_TELEMETRY', '1')
 jax.config.update('jax_enable_x64', True)
-jax.config.update(
-  'jax_pjrt_client_create_options',
-  f'ml_framework_name:PyTorch/XLA2;ml_framework_version:{"v0.0.1"}'
-)
+old_pjrt_options = jax.config.jax_pjrt_client_create_options
+
+try:
+  jax.config.update(
+    'jax_pjrt_client_create_options',
+    f'ml_framework_name:PyTorch/XLA2;ml_framework_version:{"v0.0.1"}'
+  )
+  xla_bridge._clear_backends()
+  jax.devices()  # open PJRT  to see if it opens
```

Comment:
This could be a problem in the future. If you import `torch_xla2` while another process holds the device, your second process may crash even before it tries to run anything. It could also be a problem if we add tests that create a child process to test JAX initialization for the same reason, which I may do to test `torch.distributed` init.

Is JAX going to support this option on GPU (or at least not crash if this option is given on GPU) in the near future? If so, I think this is an okay hack for now and we can fix it later.

Reply:
Yes. PJRT for CUDA need to take those args and ignore. PJRT for CPU already does that.